### DISCUSSION TITLE
Calendar: enable specific dates giving them a different style and a tooltip

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#2120] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Hello, among examples of calendar I see:
- [Enable only specific Dates](https://fomantic-ui.com/modules/calendar.html#enable-only-specific-dates) that disables all dates and only enables specific dates
- [Highlighting specific Days](https://fomantic-ui.com/modules/calendar.html#event-dates) that marks specific days as events, giving them a different cell style and a tooltip message.

I need a "merge" of them, a way to disable all dates and enable only specific dates giving them style and a tooltip message.

Is it possible to have it?

Here is my playground: https://jsfiddle.net/u6yo7erp/

**Comment by lubber-de** [comment, marked as answer]
The way you used it is IMHO already the best way to do it. Infact `enabledDates` already supports an object structure, so you can declare all your dates at a central place, but provide that to both, eventDates and enabledDates.
See https://jsfiddle.net/lubber/g5y0cetb/3/
